cluster/saltbase/salt/etcd/etcd.manifest

```diff
@@ -0,0 +1,45 @@
+{% if grains.etcd_servers is defined -%}
+  {% set etcd_server = "" + grains.etcd_servers -%}
+{% else -%}
+  {% set ips = salt['mine.get']('roles:kubernetes-master', 'network.ip_addrs', 'grain').values() -%}
+  {% set etcd_server = "" + ips[0][0] -%}
+{% endif -%}
+
+version: v1beta2
+id: etcd-server
+containers:
+  - name: etcd-container
+    image: kubernetes/etcd:2.0.1
+    command: [
+              "/usr/local/bin/etcd",
+	      "--addr",
+	      "{{etcd_server}}:4001",
```

Comment:
I continue to assert that this is brain damage, but it's fine for now.  When we go to multi-master we should re-evaluate.